src/styles/blocks/button.scss

```diff
@@ -0,0 +1,93 @@
+.button {
+  display: block;
+
+  margin: 0 auto;
+  height: 56px;
+  border-radius: 40px;
+  border: none;
+
+  background: #fff;
+
+  font-family: inherit;
+  text-decoration: none;
+  text-align: center;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 56px;
+  color: $mainColor;
+
+  transition: box-shadow 0.2s, width 0.3s;
+
+  cursor: pointer;
+
+  &--link {
+    width: 280px;
+
+    &:hover {
+      width: 273px;
+      transition: box-shadow 0.2s, width 0.3s;
```

Comment:
don't add transition on hover, Sasha told you about it in his review, now it's not smooth

https://user-images.githubusercontent.com/106310796/197962342-10991388-d2da-4ceb-aa60-2af14ecbf598.mov